addons/backpacks/anarc164/CfgVehicles.hpp

```diff
@@ -14,5 +14,6 @@ class TFAR_anarc164: TFAR_Bag_Base {
     tf_dialog = "anarc164_radio_dialog";
     tf_subtype = "airborne";
     tf_dialogUpdate = "[""%1""] call TFAR_fnc_updateLRDialogToChannel;";
+    tf_halfDuplexOverride = 0; //0 disables half-duplex override, everything greater than 0 enables it
```

Comment:
Alright, I'll push a change to add it to `TFAR_Bag_Base` as well once I'm home. I assume you mean adding it [here](https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/backpacks/CfgVehicles.hpp#L10)?